src/OpenTelemetry.Instrumentation.AspNetCore/OpenTelemetryBuilderExtensions.cs

```diff
@@ -39,6 +39,8 @@ public static class OpenTelemetryBuilderExtensions
                 throw new ArgumentNullException(nameof(builder));
             }
 
+            // Asp.Net Core is not instrumented with ActivitySource, hence
```

Comment:
@cijothomas Question... if we listen for string.Empty in theory we'll get activities from a bunch of legacy DiagnosticSources flowing through our listener, do we need to filter out the stuff we don't care about somewhere?

Reply:
You are right. This will be followed up.